### PR TITLE
Updated limit to 25MB (The new limit of Discord)

### PR DESCRIPTION
**Pull request by Sayrix**

### CODE REVIEW

**Comment by DisboxApp** [comment]
Hey! This is a great idea, didn't know they changed the limit.

It seems like they changed it to 25MB though, not 24 - why not go for 25?

**Comment by DisboxApp** [comment]
> To prevent errors

Not sure I understand. Can you elaborate?

**Comment by Sayrix** [comment]
To be sure not to exceed 25MB it is better to set the limit to 24.99MB for example

**Comment by DisboxApp** [comment]
> To be sure not to exceed 25MB it is better to set the limit to 24.99MB for example

Yeah, So I would do `25 * 1000 * 999`, but you did 24 * 1000 * 999`

**Comment by enygmate** [comment]
> > To be sure not to exceed 25MB it is better to set the limit to 24.99MB for example
> 
> Yeah, so I would do `25 * 1000 * 999`, but you did 24 * 1000 * 999`

Technically `26.2*1000*999` makes 26173800 bytes which is ~24.96Mb so it would still fit in up to 26.2.
26.3 would be too large though.

File: src/disbox-file-manager.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { sha256 } from 'js-sha256';
 
 const SERVER_URL = 'https://disboxserver.azurewebsites.net';
 export const FILE_DELIMITER = '/';
-const FILE_CHUNK_SIZE = 8 * 1000 * 999 // Almost 8MB
+const FILE_CHUNK_SIZE = 24 * 1000 * 999 // Almost 8MB
```
Comment (DisboxApp) [inline src/disbox-file-manager.js:6]: The comment `Almost 8MB` should also be changed

**Comment by DisboxApp** [comment]
So I tested this change and it seems like the webhook API still hasn't received this change. Trying to increase it above 8MB (tried even 20MB to make sure we're not passing 25) status code 413 with the following error: `{"message": "Request entity too large", "code": 40005}`

Tried this both in the default API version (currently v6) and in v9, with the v10 API returning `{"message": "Invalid API version", "code": 50041}`.

Unless someone has better luck with this and figures out how to use the increased limit, it appears this won't work for now :(

**Comment by enygmate** [comment]
> So I tested this change and it seems like the webhook API still hasn't received this change. Trying to increase it above 8MB (tried even 20MB to make sure we're not passing 25) status code 413 with the following error: `{"message": "Request entity too large", "code": 40005}`
> 
> Tried this both in the default API version (currently v6) and in v9, with the v10 API returning `{"message": "Invalid API version", "code": 50041}`.
> 
> Unless someone has better luck with this and figures out how to use the increased limit, it appears this won't work for now :(

Well that is weird, it works for me
image

**Comment by DisboxApp** [comment]
> Well that is weird, it works for me image

Do you have Nitro?

**Comment by enygmate** [comment]
> Do you have Nitro?

Yes, but this is sent using a webhook not my account.
I used DiscoHook for this example but uploading a file with curl works too

**Comment by DisboxApp** [comment]
Because webhooks are linked to your account, I believe they have the same limit.

> > Do you have Nitro?
> 
> Yes, but this is sent using a webhook not my account. I used DiscoHook for this example but uploading a file with curl works too

**Comment by DisboxApp** [comment]
> I don't have nitro yet it works

Does the PR work (or through a webhook otherwise) or just uploading files directly?

**Comment by mmccall0813** [comment]
Any updates on this?

**Comment by Z4ckMH** [comment]
> Any updates on this?

same..

**Comment by dave9123** [comment]
Is there somewhere I can test this?

**Comment by DisboxApp** [comment]
Sorry for the delay. Wasn't sure what to do with this PR since it didn't work for me but worked for other people.
Seems like this change rolled out completely now! (If someone still didn't get it, please comment here or open an issue about it!)
I updated the limit to be even closer to 25MB and tested it.
Looks good to me!

<sup>~Sorry for the bad commit message, I think I can only commit to the patch using Github's online editor which i'm not very familiar with.~</sup>

**Comment by dave9123** [comment]
You can use git
________________________________

**Pull request merged**